playbooks/files/plugins/maas_common.py

```diff
@@ -43,7 +43,7 @@
 # v2 keystone auth is still necessary until RPCR switches over to v3 auth
 
 
-OPENRC = '/root/openrc-maas'
+OPENRC = '/root/openrc'
```

Comment:
this is using the openrc file from osa instead of trying to carry our own version. This ensures we're not carrying useless code.